microcosm_flask/swagger/definitions.py

```diff
@@ -120,21 +121,53 @@ def iter_definitions(definitions, operations):
         yield get_response_schema(func)
 
 
-def build_path(operation, ns):
+def build_path_for_integer_param(ns, operation, arguments):
```

Comment:
A test of this directly would be useful, especially to see the structure of the outputted value